members/README.md

```diff
@@ -114,3 +114,26 @@ Creates new request for member introduction
 - **Error Response:**
     - **Code:** 500
         - **Content:** `{ 'statusCode': 500, 'error': 'Internal Server Error', 'message': 'An internal server error occurred' }`
+## Patch /members/archiveMembers/:username
+
+Add members/ users to Archive
+
+- **Params**\
+   _Required_: `username=[string]`
+- **Query**\
+   None
+- **Body**\
+   `{reason: <string>}`
+- **Headers**\
+   None
+- **Cookie**\
+   None
+
+  - **Success Response:**
+  - **Code:** 204
+    - **Content:**
+      None
+
+- **Error Response:**
+  - **Code:** 500
+    - **Content:** `{ 'statusCode': 500, 'error': 'Internal Server Error', 'message': 'An internal server error occurred' }`
```

Comment:
what about 400 , 401 , 404 , 409 ?

Reply:
since this is a authorized route 401 should be there. what if someone tries to update a user that doesn't exist in the db wont that throw 404 currently ?

Reply:
yes, it is throwing 404 currently if someone tries to update a user that doesn't exist in the db,
Since this PR is linked to ticket https://github.com/Real-Dev-Squad/website-backend/issues/1214, so if the reason is not present or an empty API endpoint is supposed to throw a status 400 bad request with the response

Reply:
> yes, it is throwing 404 currently if someone tries to update a user that doesn't exist in the db, Since this PR is linked to ticket [Real-Dev-Squad/website-backend#1214](https://github.com/Real-Dev-Squad/website-backend/issues/1214), so if the reason is not present or an empty API endpoint is supposed to throw a status 400 bad request with the response

`yes, it is throwing 404 currently if someone tries to update a user that doesn't exist in the db`. then shouldn't we add 404 here  ?

Reply:
No, we require status `400` if the body is empty, and `404` if the user already exists, both status are working as expected

Reply:
It is not part of this ticket,, Please ask the guy who worked on making the `assigning users/ members archived` feature!

Reply:
Improving our documentation will benefit everyone including those who are working on it and the person responsible for deprecating this route. If it requires minimal effort to add those codes, I strongly recommend we do so.

Reply:
I agree, If it could be a minimal change then I already have done it, but the whole members API contract is missing - mentioned the list below for your reference, 
I think someone should take a separate ticket to add all the missing API contracts of members separately Then it could be helpful.

This ticket is already exceeded the ETA, kindly approve this if all your queries get resolved regarding this ticket,
Can take this up after working on the current ticket

**Missing API Contracts:**
`/members/moveToMembers/${nonSuperUser.username}` // Should return 401 if user is not a super_user
`/members/archiveMembers/${userDoesNotExists.username}` // Should return 404 if user doesn't exist
`/members/archiveMembers/${userToBeArchived.username}` // Should archive the user and return 204
`/members/archiveMembers/${userAlreadyArchived.username}` // Should return 400 if user is already archived
`/members/moveToMembers/${nonSuperUser.username}`  // Should return 401 if user is not a super user
and more..